packages/accounts/src/light-account/types.ts

```diff
@@ -0,0 +1,6 @@
+import { z } from "zod";
+import { LightAccountProviderConfigSchema } from "./schema.js";
+
+export type LightAccountProviderConfig = z.input<
```

Comment:
What's the difference compared to doing z.infer?

Reply:
`z.input` allows for setting a default param in our schema and generating types that keep that param as optional. `z.infer` is the same as `z.output` which makes the optional params with defaults required